build.gradle

```
@@ -41,6 +41,7 @@ dependencies {
     testImplementation 'org.mockito:mockito-junit-jupiter:5.3.1'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'com.h2database:h2:2.1.214'
+    testImplementation 'com.github.tomakehurst:wiremock:3.0.0-beta-9'
```

```suggestion
    testImplementation 'com.github.tomakehurst:wiremock:3.0.0-beta-10'
```
вже новіша версія встигла вийти. 
тут не принципово, але на майбутнє, то є гарна звичка перевіряти наявність свіжої версії